schema.graphql

```diff
@@ -33,6 +33,13 @@ type BibleVersion {
   machineTranslation: Boolean
 }
 
+type LocalizedBookName {
+  id: ID!
+  name: String! @search(by: [term, exact])
+  book: BookReference!
```

Comment:
Do we need a `@hasinverse` decorator in the `BookReference` to ensure the two-way link?

Reply:
My thought was no doing a two-way link so that we didn't need to include localization information as part of the reference chain. The pattern would be to search for the Bible version and the localized book names that accompany it. Which I think means I need to add an `@search ` in there for the version field.

Reply:
Two way links are now included throughout.